src/bmpcodec.c

```diff
@@ -738,7 +750,7 @@ gdip_read_BITMAPINFOHEADER (void *pointer, ImageSource source, BITMAPV5HEADER *b
 		bmi->bV5ClrUsed = 0;
 		bmi->bV5ClrImportant = 0;
 
-		return Ok;
+		goto fill_rgb_masks;
```

Comment:
This looks kind of useless. There will never be masks since there's no compression and hence no BI_BITFIELDS (`bV5Compression` should be set to `BI_RGB` for clarity).  And it will never be 16-bit since the allowed values for bcBitCount are only 1, 4, 8, and 24 (https://msdn.microsoft.com/en-us/library/windows/desktop/dd183372(v=vs.85).aspx).